pkg/es/writer/esBulkHandler.go

```diff
@@ -318,3 +319,56 @@ func PostBulkErrorResponse(ctx *fasthttp.RequestCtx) {
 	responsebody["status"] = 400
 	utils.WriteJsonResponse(ctx, responsebody)
 }
+
+func ProcessDeleteIndex(ctx *fasthttp.RequestCtx, myid uint64) {
+	inIndexName := utils.ExtractParamAsString(ctx.UserValue("indexName"))
+
+	convertedIndexNames, indicesNotFound := deleteIndex(inIndexName, myid)
+
+	if indicesNotFound == len(convertedIndexNames) {
+		ctx.SetStatusCode(fasthttp.StatusNotFound)
+		response := make(map[string]interface{})
+		final := make(map[string]interface{})
+		var items = make([]interface{}, 0)
+		items = append(items, *utils.NewDeleteIndexErrorResponseInfo(inIndexName))
```

Comment:
So `items` will just have one element? Do you need to iterate over the `convertedIndexNames`?